modules/ROOT/pages/bolt/message.adoc

```diff
@@ -282,6 +282,18 @@ There are three different kinds of messages:
 | `extra::Dictionary(user_agent::String, scheme::String, routing::Dictionary(address::String))`
 | initialize connection (replaces <<messages-init, `INIT`>> of v1 & v2)(`routing::Dictionary(address::String)` added in v4.1)
 
+| <<messages-logon, `LOGON`>>
+| `6A`
+| Request
+| `scheme::String ...`
```

Comment:
Does `...` imply that more parameters can/should be passed? How does the reader know what they are?

Reply:
Yes it does it explains what should be sent in the message doc, as it depends what you have configured the server to use: `** Further entries in the message are passed to the implementation of the chosen authentication scheme.`